### ISSUE TITLE
Running code result in size() error

### ISSUE THREAD

**Issue by gree303** [tricoders/texture-synthesis#1]
After downloading & installing "Processing v3" for Linux I ran into the following issue.
As soon I hit "start" I get the following error: [Error](http://s15.postimg.org/x6t0p012j/error_size.png)
I suspect it's another processing version, but I'm not sure about this...
I've found the following in the reference:

_"As of Processing 3, to run a sketch at the full dimensions of a screen, use the fullScreen() function, rather than the older way of using size(displayWidth, displayHeight)."_

Do you have some details about this? thx

**Comment by clayheaton** [comment]
Yeah, the size is calculated dynamically in the code, which is prohibited in Processing v3. You can either change the `size()` call to be just two numbers or you can download and run the sketch in Processing v2. I don't have time to revisit/fix it right now, but will try to do so before long.

**Comment by gree303** [comment]
thanks. just got v2, works fine. It's GNU license, so to get it right if I pack this into any other package it has to be open source too? In case of any further restrictions please let me know.
